lib/perl/Genome/FeatureList/Command/Merge.pm

```diff
@@ -188,14 +188,20 @@ sub _find_convertible_reference {
         my $target = $_; all { $target->contains($_) || $available_conversions{$_->id}{$target->id} } @references;
     } @target_references;
 
-    if (scalar(@convertible_references) > 1) {
-        $class->_die_with_multiple_candidate_references(
-            'The references of the input feature-lists can be converted to multiple references:',
-            @convertible_references,
-        );
+    if (scalar(@convertible_references) < 2) {
+        return $convertible_references[0];
     }
 
-    return $convertible_references[0];
+    #prefer a reference from our original query set
+    my @preferred_references = grep { my $target = $_; any { $_ eq $target } @_ } @convertible_references;
```

Comment:
`@_` could be `@references`?  If you're worried about `@references` changing I would probably save `@_` again at beginning of function rather than use `@_` anywhere besides first couple lines of function.